modules/ting_fulltext/ting_fulltext.module

```diff
@@ -23,6 +23,14 @@ function ting_fulltext_menu() {
     'type' => MENU_CALLBACK | MENU_VISIBLE_IN_BREADCRUMB,
   );
 
+  $items['ting/fulltext/ajax/%'] = array(
```

Comment:
Is there a reason the path for this does not follow the other path declared by the module e.g. 'ting/object/%ting_fulltext_object/review`?

Also: If you use `%ting_fulltext_object` as the parameter name like we do with the path above `ting/object/%ting_fulltext_object/fulltext` then our loader `ting_fulltext_object_load()` will be called automatically and the result passed as the argument to `ting_fulltext_get_review()` and you do not have to call it yourself.